CrystiliumMod.csproj

```diff
@@ -16,4 +16,6 @@
     <None Include="Content\Gores\Crystal_Archer_Gore_1.png" />
     <None Include="Content\Gores\Crystal_Archer_Gore_2.png" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+  </ItemGroup>
```

Comment:
Please remove this - looks good otherwise. The using declaration clean-up in `SpawnCondition.cs` was a little out-of-scope, but I don't mind.